packages/terra-demographics-banner/src/_sharedObjects.jsx

```diff
@@ -72,7 +72,7 @@ const applicationIdentifiers = (props) => {
   return null;
 };
 
-module.exports = {
+export {
```

Comment:
Now that webpack is processing the imports, everything it processes much be using import/export syntax